jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-kubeadm-gce.yaml

```diff
@@ -0,0 +1,51 @@
+- job-template:
+    name: 'kubernetes-e2e-kubeadm-{suffix}'
```

Comment:
Please follow the `bootstrap-ci.yaml` for these jobs. 

Anternatively:
* Follow `bootstrap-ci-repo.yaml` to clone k8s.io/kubernetes-anywhere and create a `ci-kubernetes-e2e-kubeadm-gce` job (or something like it)
* Delete `kubernetes-anywhere-e2e-runner.sh`
* Add the E2E_OPT args to your `jobs/ci-kubernetes-e2e-kubeadm-gce.sh` script.